### PR TITLE
PEP8 in sims #884 

### PR DESCRIPTION
**Pull request by theaverageguy**
Fixes https://github.com/nipy/dipy/issues/884

### CODE REVIEW

File: dipy/sims/tests/test_phantom.py [hunk 1]
```diff
@@ -66,7 +68,7 @@ def test_add_noise():
     N = 50
     S0 = 100
 
-    options = dict(func=f,
+    options=dict(func=f,
```
Comment (arokem) [inline dipy/sims/tests/test_phantom.py]: This was actually fine before, I believe.

File: dipy/sims/voxel.py [hunk 2]
```diff
@@ -298,16 +298,14 @@ def multi_tensor(gtab, mevals, S0=100, angles=[(0, 0), (90, 0)],
     sticks = _check_directions(angles)
 
     for i in range(len(fractions)):
-            S = S + fractions[i] * single_tensor(gtab, S0=S0, evals=mevals[i],
-                                                 evecs=all_tensor_evecs(
-                                                     sticks[i]), snr=None)
+        S = S + fractions[i] * single_tensor(gtab, S0=S0, evals=mevals[i],
+                                             evecs=all_tensor_evecs(sticks[i]), snr=None)
```
Comment (arokem) [inline dipy/sims/voxel.py]: Is this still <80 characters? 

If you need to, you can put a newline right before `snr`

**Comment by arokem** [comment]
Thanks. A few small comments.

**Comment by theaverageguy** [comment]
Fixed both.

File: dipy/sims/tests/test_phantom.py [hunk 3]
```diff
@@ -78,7 +80,7 @@ def test_add_noise():
     vol = orbital_phantom(gtab, **options)
 
     for snr in [10, 20, 30, 50]:
-        vol_noise = orbital_phantom(gtab, snr=snr, **options)
+        vol_noise = orbital_phantom(gtab, snr = snr, **options)
```
Comment (arokem) [inline dipy/sims/tests/test_phantom.py]: Sorry - missed this one before. This `=` shouldn't have whitespace around it.

**Comment by arokem** [comment]
Real close here. One more teensy comment.

**Comment by theaverageguy** [comment]
Fixing it in 5!

**Comment by arokem** [comment]
Did you get a chance to fix this one?

**Comment by arokem** [comment]
This one needs a rebase.

**Comment by arokem** [comment]
Hey @theaverageguy : mind giving this one a rebase? It's got some conflicts with other things that got merged into master since.

**Comment by arokem** [comment]
I assume this has been abandoned, but feel free to reopen if you still want to work on this.

**Pull request closed without merging**